File: PWRSyllabus.Core/Entities/Course.cs
using System.Collections.Generic;
using PWRSyllabus.Core.Enums;

namespace PWRSyllabus.Core.Entities
{
    public class Course : BaseEntity
    {
        public CourseForm CourseForm { get; set; }
        public int ECTS { get; set; }
        public double ECTSForPracticalClassees { get; set; }
        public double ECTSForTeacherStudentContact { get; set; }
        public int ZZU { get; set; }
        public int CNPS { get; set; }
        public string FormOfCrediting { get; set; }
        public SubjectCard SubjectCard { get; set; }
        public int SubjectCardId { get; set; }

        public List<Class> Classes { get; set; }
    }
}

File: PWRSyllabus.API/ClientApp/src/models/Course.ts
import Class from './Class';

export default interface Course {
    id: number;
    ECTS: number;
    ZZU: number;
    CNPS: number;
    CourseForm: string;
    FormOfCrediting: string;
    Classes: Class[];
    isSelected: boolean; // I need it, okay? no matter the cost
}

const DefaultCourse: Course = {
    id: 0,
    ECTS: 0,
    ZZU: 0,
    CNPS: 0,
    CourseForm: "",
    FormOfCrediting: "",
    Classes: [],
    isSelected: false
};

export {DefaultCourse, Course};